homeassistant/components/philips_js/device_trigger.py

```diff
@@ -43,7 +43,7 @@ async def async_attach_trigger(
     config: ConfigType,
     action: AutomationActionType,
     automation_info: dict,
-) -> CALLBACK_TYPE:
+) -> Optional[CALLBACK_TYPE]:
```

Comment:
Please `return None` explicitly. Whenever we return a value anywhere in a function all exit points must return a value explicitly.